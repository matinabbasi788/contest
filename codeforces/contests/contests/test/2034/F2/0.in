5
3 4 0
1 1 1
1 0
3 3 2
1 1
2 2
3 3 2
2 1
1 2
10 4 5
1 0
8 0
6 4
0 2
7 4
