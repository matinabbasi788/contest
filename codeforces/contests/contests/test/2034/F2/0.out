10
499122180
798595498
149736666
414854846